app/models/resource.py

```diff
@@ -98,8 +99,28 @@ class Resource(db.Model):
         back_populates='resource',
         cascade='save-update, merge, delete, delete-orphan'
     )
-    suggestions = db.relationship('Suggestion', backref='resource',
-                                  uselist=True)
+    type = db.Column(db.String(20))
```

Comment:
No good reason: "resource_suggestion" is 19 characters, and 20 is a nicer number than 19.